internal/providers/system/system.go

```diff
@@ -35,7 +38,37 @@ const (
 )
 
 func FetchBaseConfig(logger *log.Logger) (types.Config, report.Report, error) {
-	return fetchConfig(logger, baseFilename)
+	var fullBaseConfig types.Config
+	baseConfig, _, err := fetchConfig(logger, baseFilename)
+	if err != nil {
+		logger.Err("couldn't fetch base config: %v", err)
+		return types.Config{}, report.Report{}, err
+	}
+	fullBaseConfig = latest.Merge(fullBaseConfig, baseConfig)
+
+	// For merging base.d config fragments
+	baseDConfig, _, err := fetchBaseConfig(logger, "base.d")
```

Comment:
We should be merging reports, here and below and in `fetchBaseConfig`, rather than ignoring them.  They may contain warnings.